datacatalog/snippets/policytagmanager/get_iam_policy.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policytagmanager
+
+// [START data_catalog_ptm_get_iam_policy]
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	datacatalog "cloud.google.com/go/datacatalog/apiv1beta1"
+	iampb "google.golang.org/genproto/googleapis/iam/v1"
+)
+
+// getIAMPolicy prints information about the IAM policy associated with a
+// given taxonomy or policy tag resource.
+func getIamPolicy(resourceID string, w io.Writer) error {
```

Comment:
s/Iam/IAM/

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
File this one under "I spend too much time working with discovery libraries."

Done.